bobcat/bb-cumber/src/main/java/com/cognifide/qa/bb/cumber/Bobcumber.java

```diff
@@ -80,12 +87,19 @@ public Bobcumber(Class<?> clazz) throws InitializationError, IOException {
         PrintWriter writer = new PrintWriter(file, CharEncoding.UTF_8);
         writer.close();
       }
+      if (clazz.getName().equals(RERUN_FAILED_TESTS_CLASS_NAME) && !anyTestFailed()) {
+        noTestsToRerun = true;
+      }
     }
   }
 
   @Override
   public void run(RunNotifier notifier) {
-    if(storeFailedResults) {
+    if (noTestsToRerun) {
```

Comment:
you should add condition here to check whether this applies to rerun class or not. In this scenario we'll override evvry result with no tests inside, but we want it only for rerun